x-pack/filebeat/input/azureblobstorage/scheduler.go

```diff
@@ -126,29 +147,6 @@ func fetchJobID(workerId int, containerName string, blobName string) string {
 	return jobID
 }
 
-func (s *scheduler) createJobs(pager *azblob.ContainerListBlobFlatPager) ([]*job, error) {
```

Comment:
removing and making it inline because struct returned from MoveNext is in internal package and not behind interface.